test/micro/org/openjdk/bench/java/util/StringJoinerBenchmark.java

```diff
@@ -41,9 +43,17 @@
  */
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
-@Fork(jvmArgsAppend = {"-Xms2g", "-Xmx2g"})
+@Warmup(iterations = 10, time = 500, timeUnit = TimeUnit.MILLISECONDS)
+@Measurement(iterations = 10, time = 500, timeUnit = TimeUnit.MILLISECONDS)
+@Fork(value = 1, jvmArgsAppend = {"-Xms1g", "-Xmx1g"})
```

Comment:
In general I think it's prudent to default to at least 3 forks, to catch issues with run-to-run variance.

Reply:
Thanks, Claes. Will change that to 3 forks before pushing.
Any more comments? Will wait for a day, then push it.